plugins/radar-android-google-sleep/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<manifest xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.ACTIVITY_RECOGNITION" />
+    <uses-permission android:name="com.google.android.gms.permission.ACTIVITY_RECOGNITION"
+        android:maxSdkVersion="28" />
+
+    <application android:allowBackup="true">
+        <service android:name=".GoogleSleepService"
+            android:exported="false"
+            android:description="@string/google_sleep_description" />
+        <receiver android:name=".SleepReceiver"
+            android:enabled="true"
+            android:exported="true"
+            tools:ignore="ExportedReceiver" />
```

Comment:
Did you try whether this is needed? Because Android won't be able to construct the receiver in any case because it is missing the constructor parameter. If the plugin works without this XML tag, please remove it to be unambiguous.

Reply:
Yes, you are right it is not needed.